web/src/user_group_edit.js

```diff
@@ -752,15 +754,15 @@ export function add_or_remove_from_group(group, group_row) {
         user_group_edit_members.edit_user_group_membership({
             group,
             removed: [user_id],
-            success_callback,
-            error_callback,
+            success: success_callback,
```

Comment:
Success and error callback weren't being called and it wasn't noticed because the spinner was being stopped after receiving the appropriate event. I'm thinking it's still better to have this logic instead of relying on just event handling for success/error scenarios here.